atomic_reactor/plugins/add_labels_in_df.py

```diff
@@ -133,8 +133,8 @@ def __init__(self, workflow, labels=None, dont_overwrite=None,
             raise RuntimeError("equal_labels have to be list")
 
     def generate_auto_labels(self, platform: str) -> Dict[str, str]:
-        # build date
-        dt = datetime.datetime.utcfromtimestamp(atomic_reactor_start_time)
+        # build date, using the current time here to be > start time of koji build
+        dt = datetime.datetime.utcfromtimestamp(time.time())
```

Comment:
We definitely cannot use the start time of prebuild task as the start time here as it has to be > start time of koji build but do let me know if there is an alternate time we can use that captures build-date more accurately.

Reply:
I tried that. Somehow, it doesn't work. This `start_time` doesn't seem to have any effect on the start time of the koji build. It's always set to the time `bump_release` reserved the build.

Reply:
Oh, I see :thinking: I thought that didn't explain why it worked in OSBS 1, but it actually does. In OBS 1, `bump_release` runs in orchestrator prebuild, and `add_labels` runs in worker. That means the start_time for add_labels is after the time when bump_release ran

If Koji ignores the start_time and we don't use it in add_labels, is it still useful for anything? Maybe in case the bump_release plugin doesn't do the reservation?

Reply:
Oh I see, it's also used in add_help. But maybe it should be the same time as the label?

Reply:
hmm, I agree. If that's the case, we can remove the `start_time` and `end_time` from build metadata(since it doesn't take effect anyway) then we don't need it to be in `workflow.data` since we just need it in the prebuild task

Reply:
we want to keep metadata in koji build, those times aren't used for koji build 'started' , 'completed' which are irrelevant anyway

when user downloads metadata json, those times have to be there

and you can use real pipeline start time , as its relation to koji build itself is irrelevant

Reply:
ok. For the build-date label, should we use start time as well or start time of koji build?

Reply:
Same time we use in the other plugins IMO

Reply:
hmm, in that case we'll have to change the test case